controllers/articles-controllers.js

```diff
@@ -18,3 +18,13 @@ exports.getArticle = (req, res, next) => {
     .catch(next);
 }
 
+exports.patchArticle = (req, res, next) => {
+    const { article_id } = req.params
+    const { inc_votes } = req.body
+    return updateArticle(article_id, inc_votes)
```

Comment:
Nice consistent use of destructuring throughout makes for very clear readable code 📖